src/main/java/com/agilethought/internship/university/service/UniversityService.java

```diff
@@ -1,13 +1,12 @@
 package com.agilethought.internship.university.service;
 
-import com.agilethought.internship.university.domain.CreateCourseRequest;
-import com.agilethought.internship.university.domain.CreateCourseResponse;
-import com.agilethought.internship.university.domain.CourseResponse;
+import com.agilethought.internship.university.domain.*;
 
 import java.util.List;
 
 public interface UniversityService {
 
     CreateCourseResponse createCourse(CreateCourseRequest request);
     List<CourseResponse> getCourses();
+    UpdateCourseResponse Updatecourse(UpdateCourseRequest request, String id);
```

Comment:
Remember, method names are in camelcase with first letter in lowercase, updateCourse